config/locales/de.yml

```diff
@@ -397,17 +837,55 @@ de:
         einen Platz im Programm gefunden hat. Sie findet am %{date}
         um %{time} in %{room} statt.
       subject: Deine Einreichung zur %{conference} am %{date} um %{time}
-  error_signing_in: E-Mailadresse oder Passwort ungültig.
+  emails_module:
+    add_mail_template: E-Mail-Vorlage hinzufügen
+    add_mail_template_hint: Fügen Sie eine neue E-Mail-Vorlage hinzu.
+    edit_template: E-Mail-Vorlage bearbeiten
+    edit_template_hint: Bearbeiten Sie diese E-Mail-Vorlage.
+    edit_template_title: E-Mail-Vorlage "%{template}"
+    error_failed_setting_notification: 'Fehler beim Festlegen der Standardbenachrichtigung: %{ex}.'
+    error_missing_conference_email: 'E-Mails können nicht gesendet werden: Bitte geben Sie eine E-Mail-Adresse für diese Konferenz an.'
+    error_missing_notification_text: |
+      Kein Benachrichtigungstext vorhanden. Bitte ändern Sie den Standardtext für Ihre
+      vor dem Akzeptieren / Ablehnen von Ereignissen.
+    error_missing_speaker_email: 'E-Mails können nicht gesendet werden: Nicht alle Sprecher haben E-Mail-Adressen.'
+    error_state_update: Status kann nicht aktualisiert werden.
+    error_state_update_ex: 'Status kann nicht aktualisiert werden: %{ex}.'
+    error_unnotifiable_state: Ereignis nicht in einem Status, der eine Benachrichtigung auslöst.
+    filters:
+      all_speakers_in_confirmed_events: Alle Sprecher an allen bestätigten Veranstaltungen
+      all_speakers_in_scheduled_event: Alle Sprecher an allen Veranstaltungen, die mit Raum und Anfangszeit im Programm gesetzt sind
+      all_speakers_in_unconfirmed_events: Alle Sprecher an allen unbestätigten Veranstaltungen
+    mail_subject: Subject
+    mail_template_hint: |
+      Verwenden Sie diese E-Mail-Vorlage, um eine E-Mail an alle Personen zu senden, die an einem
+      Event dieser Konferenz beteiligt sind.
+    new_mail_template: Neue Mailvorlage
+    notice_event_updated: Das Event wurde erfolgreich aktualisiert.
+    notice_mails_delivered: E-Mails wurden ausgeliefert.
+    notice_mails_queued: E-Mails wurden an den Ticket-Server übergeben.
+    notice_template_destroyed: E-Mail-Vorlage wurde erfolgreich gelöscht.
+    notice_template_updated: E-Mail-Vorlage wurde erfolgreich aktualisiert.
+    notice_transport_need_added: Transportbedarf wurde erfolgreich hinzugefügt.
+    placeholders_hint: |
+      Die folgenden Platzhalter können jeweils im Abschnitt "Inhalt" verwendet werden
+      Vorlage.
+    send_mail: E-Mail senden
+    template_subject: 'Betreff: %{subject}'
+  error_signing_in: E-Mail-Adresse oder Passwort ungültig.
   errors:
     format: "%{attribute} %{message}"
     messages:
-      accepted: muss akzeptiert werden
+      accepted: muss accepted sein
```

Comment:
better "muss **angenommen** sein"?

Reply:
I just saw line 1006 (key `print_accepted_cards` and following), where this wording re-appears. So change all or leave all.

Reply:
Same is valid for "rejected" five comment above and "rejecting" in `reject_event` (line 1016).